src/test/transaction_tests.cpp

```diff
@@ -326,9 +326,7 @@ SetupDummyInputs(CBasicKeyStore& keystoreRet, CCoinsViewCache& coinsRet)
 BOOST_AUTO_TEST_CASE(test_basic_joinsplit_verification)
 {
     // We only check that joinsplits are constructed properly
-    // and verify properly here. libsnark tends to segfault
-    // when our snarks or what-have-you are invalid, so
-    // we can't really catch everything here.
+    // and verify properly here.
```

Comment:
maybe replace or remove the second _properly_ here?

"We only check that joinsplits are constructed properly and verify them here."